packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -467,6 +467,14 @@ export default class PciInstancesAddController {
           networkId: get(this.selectedPrivateNetwork, 'id'),
         },
       ];
+      if (!this.isPrivateMode()) {
+        this.instance.networks = [
+          ...this.instance.networks,
+          {
+            networkId: get(this.publicNetwork, 'id'),
+          },
+        ];
+      }
```

Comment:
suggestion: you can add else clause for case -> this.instance.networks = [
        {
          networkId: get(this.selectedPrivateNetwork, 'id'),
        },
      ];

Reply:
ignore it

Reply:
no, its not like this, I am updating the `this.instance.networks` conditionally.